torch/_utils.py

```diff
@@ -141,10 +141,33 @@ def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, bac
     return tensor
 
 
+_sparse_tensors_to_validate = []
```

Comment:
Is it possible to get rid of the global state here? Could cause issues when threading or leave things in an invalid state.

Reply:
Not that I can think of. I don't know enough about Python threading and GIL to understand the dangers. Do you think I should guard this with some mutex in _legacy_load() in serialization.py? That would get really ugly.

Reply:
Can you somehow put it inside `_legacy_load` and pass it around?

Reply:
I cannot think of a way to do that.